projects/ad400x-st/src/common/common_data.c

```diff
@@ -73,5 +73,6 @@ struct ad400x_init_param ad400x_ip = {
 		.number = GPIO_CNV,
 		.platform_ops = GPIO_OPS,
 	},
+	.ref_voltage_mv = AD400X_ADC_REF_VOLTAGE,
```

Comment:
after adding ref_voltage_mv in iio_driver, please add these changes in the project commit file (this commit should not be needed if the changes are performed in the other commits accordingly)

Reply:
ok,